english/super-man/save-the-word.md

```diff
@@ -6,5 +6,6 @@ The trangle underwear is mounted on it, shining a great "V"
 As the most glory superman, you decide to save the world. 
 There are Sooooo ~~~ many troubles in or out of our planet.
 They are all making a unique voice:  calling for a Man like you!
-Do you:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.